filtro/views.py

```diff
@@ -426,7 +426,8 @@ def listar_resultados(request, idfiltro):
             'total_classificados': total_classificados,
             'total_documentos': total_documentos,
             'total': total,
-            'classe': classe
+            'classe': classe,
+            'mostra_lda': total >= settings.MININUM_DOC_COUNT_LDA,
```

Comment:
Desta forma o sistema irá mostrar o botão caso o total de documentos dentre todas as classses seja maior que o mínimo de documentos não classificados, que são utilizados para fazer lda.
O algoritmo só roda para os documentos não classificados, para ajudar o usuário a encontrar agrupamentos de palavras que não conhece, então, só roda para documentos em que a classe é None.

Eu alteraria para algo como:
```suggestion
            'mostra_lda': m_filtro.saida_lda is not None
```

Reply:
Perfeito!